common/list/lists-fr.html

```diff
@@ -411,3 +411,16 @@ <h2 id="dl-inline">Liste descriptive en ligne</h2>
 	&lt;dt>Terme&nbsp;:&lt;/dt>
 	&lt;dd>Description courte&lt;/dd>
 &lt;/dl></code></pre>
+
+<h2 id="compact">Liste avec classe utilitaire <code>compact</code>.</h2>
```

Comment:
Just to add a little bit of context of use.
```suggestion
<h2 id="compact">Liste avec classe utilitaire <code>compact</code>.</h2>
<p>Liste compacte utilisé principalement pour des liens navigables sur des pages de navigation.</p>
```